doc/man1/openssl-dhparam.pod.in

```diff
@@ -28,7 +28,11 @@ B<openssl dhparam>
 
 =head1 DESCRIPTION
 
-This command is used to manipulate DH parameter files.
+This command is used to manipulate DH and DSA parameter files.
```

Comment:
Maybe leave the DSA bit out? The main purpose of this app is to generate DH params?

Reply:
okay I will leave out the DSA bit.

Reply:
udpated commit pushed.